OrbitGl/DataManager.h

```diff
@@ -60,6 +60,9 @@ class DataManager final {
   [[nodiscard]] const UserDefinedCaptureData& user_defined_capture_data() const {
     return user_defined_capture_data_;
   }
+  [[nodiscard]] UserDefinedCaptureData& user_defined_capture_data() {
```

Comment:
Again, I'd prefer to give this method a more explicit name, e.g. containing "Mutable".

Reply:
Done.